README.md

```diff
@@ -547,7 +547,7 @@ const uiSchema = {
 };
 ```
 
-Unlike other types, array's do not support the `required` property.  You can however instead specify the minimum number of items the user must select with the `minItems` property.
+Care should be taken when using the `required` property with arrays.  An empty array is sufficient to pass that validation check.  If you wish to ensure the user populates the array, you can specify the minimum number of items the user must select with the `minItems` property.
```

Comment:
Unfortunately there wasn't a text area widget section that I could see so I created one .... but it isn't right.  I will have to look at it over the weekend.  

What's checked in is passing, but it's not using the widget.

I've tried the change:

    const uiSchema = {"ui:widget": "textarea"};

    it("should handle an empty string change event", () => {
      const {comp, node} = createFormComponent({schema: {
        type: "string",
      }, uiSchema});

... which seems to me like it should work but that fails for reasons that currently escape me.  I'll need to look at it over the weekend.  Got to head off now though!